src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.html

```diff
@@ -251,3 +208,124 @@
     </ng-container>
   </cd-modal>
 </div>
+
+<ng-template #nonMultiClusterTemplate>
+  <ng-container *ngIf="!loading; else loadingTemplate">
+    <ng-container *ngIf="!setupCompleted else exportTokenTemplate">
+      <ng-container *ngTemplateOutlet="reviewTemplate"></ng-container>
+    </ng-container>
+  </ng-container>
+</ng-template>
+
+<ng-template #loadingTemplate>
+  <ng-container *ngTemplateOutlet="progressTemplate"></ng-container>
+</ng-template>
+
+<ng-template #progressCompleteTemplate>
+  <div *ngIf="isMultiClusterConfigured; else exportTokenTemplate">
+    <div class="text-center text-success"
+         i18n>
+      Multi-site replication setup is complete.
+    </div>
+  </div>
+</ng-template>
```

Comment:
We should pass this message to progress component only
`<cd-progress helperText="Multi-site replication setup is complete"  status="finished"/>`

Reply:
This is required in this component itself as we are showing different templates on the same page based on different conditions